osf/models/institution.py

```diff
@@ -18,18 +18,37 @@ class Institution(Loggable, base.ObjectIDMixin, base.BaseModel):
     # /TODO DELETE ME POST MIGRATION
 
     # TODO Remove null=True for things that shouldn't be nullable
-    banner_name = models.CharField(max_length=255, null=True, blank=True)
+
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True, default='')
+
+    # TODO Could `banner_name` and `logo_name` be a FilePathField?
+    # Both banner_name and logo_name are the name of the file under the `banners/` or `shields/` folder in
+    # `website/static/img/institutions/`. They cannot be null or empty, otherwise OSF dashboard page and institution
+    # project page won't finish loading.
+    banner_name = models.CharField(max_length=255, blank=True, default='')
```

Comment:
Both `banner_name` and `logo_name` must be a non-empty string. I remove `null=True` but set default as empty string. The banner and logo files of the name must exist for each institutions.

I had thought `blank` is for non-empty at first but it is not at database level.